src/main/java/uk/gov/ons/census/fwmt/rmadapter/canonical/CanonicalJobHelper.java

```diff
@@ -190,8 +191,19 @@ private static void createIndefinitePause(CancelFieldWorkerJobRequest cancelJobR
   }
 
   public static UpdateFieldWorkerJobRequest newUpdateJob(ActionInstruction actionInstruction) {
+    ActionUpdate actionUpdate = actionInstruction.getActionUpdate();
+
     UpdateFieldWorkerJobRequest updateJobRequest = new UpdateFieldWorkerJobRequest();
     updateJobRequest.setActionType("update");
+    updateJobRequest.setId(UUID.fromString(actionUpdate.getCaseId()));
```

Comment:
should be updateJobRequest.setCaseId(...) Ive update datamap, the change needs to happen in canonical. Im keeping it consistent with Action Request

Reply:
Have update this and have updated canonical.